core/src/i18n/pl.yaml

```diff
@@ -128,6 +128,25 @@ settings:
   confirmations: Potwierdzenia
   dontConfirmDelete: Usuwanie bez potwierdzenia
   language: Język
```

Comment:
A sanity check: In the English version, language is in `common:`. Are we sure it's supposed to be here, and not 6 lines farther?
![image](https://user-images.githubusercontent.com/53601578/128851443-fcdef945-7bca-4517-a6c6-edf71b5fcb3b.png)

Reply:
As discussed via PM, it's positioned correctly in the `pl.yaml` file, and was misplaces in the `en.yaml` file, but has already been fixed. 
![image](https://user-images.githubusercontent.com/53601578/129014828-8dba1d51-01fb-49f5-8a81-5ded0b953fa4.png)